docs/t-sql/functions/serverproperty-transact-sql.md

```diff
@@ -59,7 +59,7 @@ Is an expression that contains the property information to be returned for the s
 |EditionID|EditionID represents the installed product edition of the instance of [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)]. Use the value of this property to determine features and limits, such as [Compute Capacity Limits by Edition of SQL Server](../../sql-server/compute-capacity-limits-by-edition-of-sql-server.md).<br /><br /> 1804890536 = Enterprise<br /><br /> 1872460670 = Enterprise Edition: Core-based Licensing<br /><br /> 610778273= Enterprise Evaluation<br /><br /> 284895786 = Business Intelligence<br /><br /> -2117995310 = Developer<br /><br /> -1592396055 = Express<br /><br /> -133711905= Express with Advanced Services<br /><br /> -1534726760 = Standard<br /><br /> 1293598313 = Web<br /><br /> 1674378470 = [!INCLUDE[ssSDS](../../includes/sssds-md.md)] or [!INCLUDE[ssSDW](../../includes/sssdwfull-md.md)]<br /><br /> -1461570097 = Azure SQL Edge Developer <br /><br /> 1994083197 = Azure SQL Edge <br /><br />Base data type: **bigint**|  
-|EngineEdition|[!INCLUDE[ssDE](../../includes/ssde-md.md)] edition of the instance of [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] installed on the server.<br /><br /> 1 = Personal or Desktop Engine (Not available in [!INCLUDE[ssVersion2005](../../includes/ssversion2005-md.md)] and later versions.)<br /><br /> 2 = Standard (This is returned for Standard, Web, and Business Intelligence.)<br /><br /> 3 = Enterprise (This is returned for Evaluation, Developer, and Enterprise editions.)<br /><br /> 4 = Express (This is returned for Express, Express with Tools, and Express with Advanced Services)<br /><br /> 5 = [!INCLUDE[ssSDS](../../includes/sssds-md.md)]<br /><br /> 6 = [!INCLUDE[ssDW](../../includes/ssdw-md.md)]<br /><br /> 8 = [!INCLUDE[ssSDSMIfull](../../includes/sssdsmifull-md.md)]<br /><br /> 9 = Azure SQL Edge (This is returned for all editions of Azure SQL Edge)<br /><br /> Base data type: **int**|  
+|EngineEdition|[!INCLUDE[ssDE](../../includes/ssde-md.md)] edition of the instance of [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] installed on the server.<br /><br /> 1 = Personal or Desktop Engine (Not available in [!INCLUDE[ssVersion2005](../../includes/ssversion2005-md.md)] and later versions.)<br /><br /> 2 = Standard (This is returned for Standard, Web, and Business Intelligence.)<br /><br /> 3 = Enterprise (This is returned for Evaluation, Developer, and Enterprise editions.)<br /><br /> 4 = Express (This is returned for Express, Express with Tools, and Express with Advanced Services)<br /><br /> 5 = [!INCLUDE[ssSDS](../../includes/sssds-md.md)]<br /><br /> 6 = [!INCLUDE[ssDW](../../includes/ssdw-md.md)]<br /><br /> 8 = [!INCLUDE[ssSDSMIfull](../../includes/sssdsmifull-md.md)]<br /><br /> 9 = Azure SQL Edge (This is returned for all editions of Azure SQL Edge)<br /><br /> 11 = Azure Synapse Serverless (on-demand)<br /><br /> Base data type: **int**|  
```

Comment:
returned for all editions of Azure SQL Edge)<br /><br /> 11 = Azure Synapse serverless SQL pool<br /><br /> Base data type: **int**|  
```